exasol_transformers_extension/deployment/language_container_deployer_cli.py

```diff
@@ -52,6 +26,8 @@ def run_deployer(deployer, upload_container: bool = True,
                   utils.DB_PASSWORD_ENVIRONMENT_VARIABLE, ""))
```

Comment:
The click Options are not yet packed into `@,add_options(create_deployer_option(utils.DB_PASSWORD_ENVIRONMENT_VARIABLE, utils.BUCKETFS_PASSWORD_ENVIRONMENT_VARIABLE)`, but I think we can only do it during reintegration. Maybe in the meantime we should copy the add_options from itde to here. This prepare this already.